jenkins/freestyle/get-sitespeed-report.sh

```diff
@@ -22,32 +35,98 @@ set -e
 : ${CONNECTION:?"The CONNECTION environment variable must be set."}
 : ${SITESPEED_BROWSER:?"The SITESPEED_BROWSER environment variable must be set."}
 
+if [ $GET_URLS_FROM_FILE == "true" ] ; then
```

Comment:
could: Should we collapse these into one env var (e.g., if you list a file, then we're going to use the urls in that file).